src/views/Login.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="max-w-lg mx-auto p-4 shadow-lg mt-6">
+    <form @submit.prevent="login" class="p-8 border rounded">
+      <div class="flex items-center text-2xl justify-center mb-4">
+        <img
+          class="w-16 h-16 mr-4 rounded-full"
+          src="@/assets/tu20logo.png"
+          alt="TU20 Logo"
+        />
+        TU20 Bot Config
+      </div>
+
+      <div>
+        <small class="font-light text-gray-500 text-xs mb-2"> Username </small>
```

Comment:
I don't remember what your justification for using the `<small>` tag was 😓 

The tag itself is useless and behaves exactly the same as `<span>` because of normalize.css. It doesn't tell the programmer any more about the component than `text-xs` does. I don't mind you keeping it here, but it's definitely strange